pkg/cli/democluster/demo_cluster.go

```diff
@@ -584,7 +592,14 @@ func (c *transientCluster) startNodeAsync(
 	serv := c.servers[idx]
 	tag := fmt.Sprintf("start-n%d", idx+1)
 	return c.stopper.RunAsyncTask(ctx, tag, func(ctx context.Context) {
+		// We call Start() with context.Background() because we don't want the
+		// tracing span corresponding to the task started just above to leak into
+		// the new server. Server's have their own Tracers, different from the one
+		// used by this transientCluster, and so traces inside the Server can't be
+		// combined with traces from outside it.
+		ctx = logtags.WithTags(context.Background(), logtags.FromContext(ctx))
```

Comment:
this means the code is assuming that the Start() call will have its own tracer, right?

Reply:
if so, maybe worth clarifying in the comment